src/catalog/postgres/builder.cpp

```diff
@@ -548,4 +589,136 @@ IndexSchema Builder::GetConstraintForeignTableIndexSchema(db_oid_t db) {
   return schema;
 }
 
+IndexSchema Builder::GetLanguageOidIndexSchema(db_oid_t db) {
+  std::vector<IndexSchema::Column> columns;
+
+  columns.emplace_back("lanoid", type::TypeId::INTEGER, false,
+                       parser::ColumnValueExpression(db, LANGUAGE_TABLE_OID, LANOID_COL_OID));
+  columns.back().SetOid(indexkeycol_oid_t(1));
+
+  // Primary
+  IndexSchema schema(columns, storage::index::IndexType::HASHMAP, true, true, false, true);
+
+  return schema;
+}
+
+IndexSchema Builder::GetLanguageNameIndexSchema(db_oid_t db) {
+  std::vector<IndexSchema::Column> columns;
+
+  columns.emplace_back("lanname", type::TypeId::VARCHAR, MAX_NAME_LENGTH, false,
+                       parser::ColumnValueExpression(db, LANGUAGE_TABLE_OID, LANNAME_COL_OID));
+  columns.back().SetOid(indexkeycol_oid_t(1));
+
+  // Unique, not primary
+  IndexSchema schema(columns, storage::index::IndexType::HASHMAP, true, false, false, true);
```

Comment:
This should be a bwtree since we may want to do range queries (e.g. select all languages in a given namespace).